modules/fedimint-dummy-server/src/lib.rs

```diff
@@ -247,8 +247,3 @@ impl Dummy {
         Dummy { cfg }
     }
 }
-
-#[derive(Debug, Clone)]
-pub struct DummyVerificationCache;
```

Comment:
Verification cache should be a part of a server. Client has no need for it. So I would delete the duplicate in common.

Reply:
Yes makes sense